### PR TITLE
OMERO_HOME

### PR DESCRIPTION
**Pull request by jburel**
Feedback from testing
See https://forum.image.sc/t/omero-python-3-5-6-milestone-for-upgrade-testing/32120/2

### CODE REVIEW

**Comment by mtbc** [comment]
Why on Earth are people setting it? https://docs.openmicroscopy.org/omero/5.5.1/sysadmins/omero-home-prefix.html is rather unambiguous.

**Comment by mtbc** [comment]
I guess I should be clearer: has the *intended* usage of `OMERO_HOME` become deprecated too? I.e. should the message be "you can continue using it if you were using it for what it's *supposed* to be for" or should we update the docs to say that it doesn't have any development use, or something else?

**Comment by jburel** [comment]
@mtbc I think we should get rid of the page. (doc PR)

**Comment by will-moore** [comment]
I guess we could fix our code to not check for ```OMERO_HOME```. Currently ```omeroweb/settings.py``` does this:
```
if 'OMERO_HOME' in os.environ:
    OMERO_HOME = os.environ.get('OMERO_HOME')
elif 'OMERODIR' in os.environ:
    OMERO_HOME = os.environ.get('OMERODIR')
else:
    OMERO_HOME = os.path.join(os.path.dirname(__file__), '..', '..', '..')
    OMERO_HOME = os.path.normpath(OMERO_HOME)
```
Should probably just check for ```OMERODIR``` and if not found then fail, as we do with ```omero config set``` etc in https://github.com/ome/omero-py/pull/14

**Comment by mtbc** [comment]
Could even warn if `OMERO_HOME` is found to be set.

File: ansible/templates/common/introduction.j2 [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {% if doc %}
-This is an example walkthrough for installing OMERO.web in a **virtual environment** using a dedicated system user. Installing OMERO.web in a virtual environment is the preferred way. For convenience in this walkthrough, we will use the **{{ omero_user }} system user** and define the main OMERO.web configuration options as environment variables.
-{% endif %}
\ No newline at end of file
+This is an example walkthrough for installing OMERO.web in a **virtual environment** using a dedicated system user. Installing OMERO.web in a virtual environment is the preferred way. For convenience in this walkthrough, we will use the **{{ omero_user }} system user** and define the main OMERO.web configuration options as environment variables. Since 5.6, a new :envvar:`OMERODIR` variable is used, you should first unset :envvar:`OMERO_HOME` (if set) before beginning the installation process.
```
Comment (joshmoore) [inline ansible/templates/common/introduction.j2]: NB: can this long line be wrapped?

Comment (jburel) [reply ansible/templates/common/introduction.j2]: The display when generated is not ideal. I tried in the past and did not get the desired outcome
I can try gain

**Comment by joshmoore** [comment]
Assuming this should be merged along with https://github.com/ome/ome-documentation/pull/2062.

**Pull request merged**